src/components/MainPage/Importers.vue

```diff
@@ -69,12 +71,16 @@ export default {
     ...mapState({
       scrapers: (state) => state.Scrapers.scrapers,
     }),
-    ...mapGetters([
-      'scrapersWithId',
-      'importers',
-    ]),
+    ...mapGetters(['scrapersWithId', 'importers']),
+  },
+  created() {
+    ipcRenderer.send('getScrapers');
+    ipcRenderer.on('getScrapers-reply', (event, scrapers) => {
+      this.set_scraper(scrapers);
+    });
   },
   methods: {
+    ...mapActions(['set_scraper']),
```

Comment:
I don't understand why the `SCRAPERS` from `israeli-bank-scrapers-core` needs to imported in `scrapers.service`, then imported by `background.js`, who is expose them through the `ipc`, and then this `Importers.vue` call them to add them to the `store`, then uses them by `state` and `getters`.

Reply:
From @Arielgordon123 : There is a conflict in `ws`, when using `israeli-bank-scrapers-core` in the `renderer` process.